fe/fe-core/src/main/java/org/apache/doris/nereids/types/DataType.java

```diff
@@ -348,7 +357,6 @@ public static DataType fromCatalogType(Type type) {
             case DATEV2: return DateV2Type.INSTANCE;
             case DATE: return DateType.INSTANCE;
             case TIMEV2: return TimeV2Type.INSTANCE;
-            case TIME: return TimeType.INSTANCE;
```

Comment:
maybe we should translate catalog type TIME to nereids' TimeV2 too, because maybe TIME type already presist in meta when create alias function